### DISCUSSION TITLE
First option active (=highlighted) by default in Combobox

### DISCUSSION THREAD

**Discussion by axelv** [tailwindlabs/headlessui#1219] [Help]
First of all, thanks for this awesome library ! Love the neat separation of style and functionality.

I'm playing around with Combobox which very wel designed. However, I'm don't get how the **active** option is chosen internally.

Can I set the first option active when the use is still typing? 
This would make more sense when building  command pallets imho.

### Example:
In [https://tailwindui.com/components/application-ui/navigation/command-palettes#component-3b1e5d9bec70736c7d1afca2585ec39e](TailwindUI/Command-Pallets) when I type "Foster" the first match is highlighted but when typing "Francis" non of the matches is highlighted.
![2022-03-09 16 40 44](https://user-images.githubusercontent.com/1982751/157476787-09356284-ab7d-4b1f-96e2-23fc8524b0e9.gif)

**Comment by axelv** [comment]
Sorry, this is exactly the same topic as https://github.com/tailwindlabs/headlessui/discussions/1178 so this can be closed

**Comment by RobinMalfait** [comment]
Hey! Thank you for your suggestion!
Much appreciated! 🙏

This pr should fix that: https://github.com/tailwindlabs/headlessui/pull/1279
